src/main/java/seedu/address/model/calendarevent/DateTime.java

```diff
@@ -12,7 +12,7 @@
 public class DateTime {
 
     public static final String MESSAGE_DATETIME_CONSTRAINTS =
-            "The valid input will be year-month-localDateTime hour:minute";
+        "The valid input will be year-month-localDateTime hour:minute";
```

Comment:
will it be year-month-day hour:minute?

Reply:
hmm looks like it hasn't been updated yet

Reply:
I think no, but don't need to change la. Cuz vig has changed.